### ISSUE TITLE
Provisioning Cluster Time

### ISSUE THREAD

**Issue by DevanB** [graphcool/prisma-cloud-feedback#46]
I've been waiting on a cluster to provision for over an hour now.

I'm unsure what data you need, so I'm here to share anything to help debug what is happening!

**Comment by gentle-noah** [comment]
@DevanB - what if you hard refresh?

**Comment by DevanB** [comment]
A hard refresh tells me that I have filled out a form and haven't "submitted" it. I'm sure that all form data has been transferred to the application though. 

After a hard refresh, I am returned to the default "New Cluster" form. 

After navigating to the Cluster dashboard, it still says my cluster is provisioning.

**Comment by gentle-noah** [comment]
have you tried deleting your cluster? What happens?

**Comment by DevanB** [comment]
I’d prefer not to delete it, in the case any of the Graphcool developers need to see the current state of it

**Comment by gentle-noah** [comment]
This seems very similar to what I'm going through at [#39](https://github.com/graphcool/prisma-cloud-feedback/issues/39) - so I was just curious to see if you would end up in the same state

**Comment by maxdarque** [comment]
I had this issue as well the first time I tried to provision a cluster. I ended up deleting it and re-provisioning it.

**Comment by iamclaytonray** [comment]
I'm running into the same issue, however, hard refresh doesn't do what you reported @DevanB. I've also tried to delete another one (Prisma ended up replicating a cluster) but still in limbo

**Comment by marktani** [comment]
Currently, the expected provisioning time is ~10 minutes. We'll decrease that significantly next week.
If your provisioning takes much longer than 10 minutes, it's safe to assume that something went wrong.

Additionally, some deletion requests are currently failing, as described here: https://github.com/graphcool/prisma-cloud-feedback/issues/50. We will fix this soon.

**Comment by iamclaytonray** [comment]
Update: Both of my clusters are now in deletion mode. Not sure if that helps with diagnosis

**Comment by gentle-noah** [comment]
@marktani - my new cluster got stuck in the provisioning state. I waited 20 minutes, refreshed and everything is working perfectly. The UI just did not update properly. Deployed, data incoming, feeling great.

**Comment by iamclaytonray** [comment]
@gentle-noah - Did you try deleting any clusters? Just curious for where I am

**Comment by DevanB** [comment]
I tried deleting that cluster that was taking a while to provision. It has been in the deleting state for over 24 hours now. Is there anything else I can do to help?

**Comment by marktani** [comment]
Provisioning a new cluster should now take ~5 seconds most of the time.
If you still have problems with "dangling" clusters, please create a new issue or reach out to me.

Thanks!
